lib/aws.py

```diff
@@ -326,6 +350,30 @@ def plan_asgs(asgs):
 
     return asg_outdated_instance_dict
 
+def plan_asgs_older_nodes(asgs):
```

Comment:
Just naming again. I like to call my function exactly what they do. This gets you a list of outdated nodes. I'd call it `get_outdated_asg_nodes` or  `get_outdated_asg_instances` since you seem to be using instances in other places in the code.

Edit : I see that you're following their convention for plan. Maybe just stay consistent with naming then. `plan_asgs_outdated_instances` instead of older_nodes